### DISCUSSION TITLE
NaN in training

### DISCUSSION THREAD

**Discussion by weiyana** [fredzzhang/upt#72] [Q&A] [answered]
Hi, thanks for your good work and your code! When I trained with one gpu on the full HICODET, or trained with 4 gpus on one subset of HICODET(5000 imgs), I always met NaN. Did you met this before? Can I fix this using some common engineering techniques? 
Thank you!

**Comment by fredzzhang** [comment, marked as answer]
Hi @weiyana,

Please refer to #71. The issue is the batch size. The number of GPUs you use times the per-GPU batch size should be at least 16.

Fred.

**Comment by weiyana** [comment]
This works for me, thx!
